Formula/shtools.rb

```diff
@@ -0,0 +1,44 @@
+class Shtools < Formula
+  desc "Spherical Harmonic Tools"
+  homepage "https://shtools.github.io/SHTOOLS/"
+  url "https://github.com/SHTOOLS/SHTOOLS/archive/v4.7.tar.gz"
+  sha256 "2d6f426dbdeb8e3a16b1cd5ea5dd5db228bdb76f438abebcccd15e43096e8d37"
+  license "BSD-3-Clause"
+  head "https://github.com/SHTOOLS/homebrew-shtools.git"
+
+  depends_on "fftw"
+  depends_on "gcc"
+
+  def install
+    ENV.deparallelize
+    system "make", "fortran"
+    system "make", "fortran-mp"
+
+    pkgshare.install "examples/fortran/", "examples/ExampleDataFiles/"
+    inreplace pkgshare/"fortran/Makefile", "../../lib", "/usr/local/lib"
```

Comment:
This looks like it should be patched upstream.

Reply:
The problem is that if decided to clone the git repo and manually compile the code yourself in your local folder, these files would not not located at `/usr/local`. These filenames/paths need to be changed when being "installed" elsewhere.

Reply:
But that's what you provide `PREFIX`/`DESTDIR` for in the make command. So the software knows where it's going to be.

Reply:
Ok. I'll refactor the Makefile to make it work that way when compiling locally.

Question: Should I close this PR, and re-open a new one when we release a minor update? Or should I just keep this open and modify it later?

Reply:
Generally people modify it later